<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Preference</class>
 <widget class="QDialog" name="Preference">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>664</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preference</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>user settings</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="playerName_label">
          <property name="text">
           <string>player name:</string>
          </property>
          <property name="buddy">
           <cstring>playerName_le</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="playerName_le"/>
        </item>
        <item>
         <widget class="QCheckBox" name="autoName_cb">
          <property name="text">
           <string>auto use original name</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="email_label">
            <property name="text">
             <string>E-mail:</string>
            </property>
            <property name="buddy">
             <cstring>email_le</cstring>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="password_label">
            <property name="text">
             <string>password:</string>
            </property>
            <property name="buddy">
             <cstring>password_le</cstring>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLineEdit" name="email_le"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLineEdit" name="password_le"/>
            </item>
            <item>
             <widget class="QPushButton" name="showPassword_pb">
              <property name="text">
               <string>&amp;Show password</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>game settings:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="corePath_label">
            <property name="text">
             <string>core game path:</string>
            </property>
            <property name="buddy">
             <cstring>corePath_le</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="corePath_le"/>
          </item>
          <item>
           <widget class="QPushButton" name="corePath_showPb">
            <property name="text">
             <string>Show...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="width_label">
            <property name="text">
             <string>width:</string>
            </property>
            <property name="buddy">
             <cstring>width_sb</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="width_sb">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximum">
             <number>999999999</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="height_label">
            <property name="text">
             <string>height:</string>
            </property>
            <property name="buddy">
             <cstring>height_sb</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="height_sb">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximum">
             <number>999999999</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Java settigns:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="memCtrl_label">
            <property name="text">
             <string>memory control:</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>269</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="minMem_label">
              <property name="text">
               <string>min memory</string>
              </property>
              <property name="buddy">
               <cstring>minMem_sb</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="minMem_sb">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximum">
               <number>999999999</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="maxMem_label">
              <property name="text">
               <string>max memory:</string>
              </property>
              <property name="buddy">
               <cstring>maxMem_sb</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="maxMem_sb">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximum">
               <number>999999999</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="javaPath_label">
            <property name="text">
             <string>Java path:</string>
            </property>
            <property name="buddy">
             <cstring>javaPath_le</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="javaPath_le"/>
          </item>
          <item>
           <widget class="QPushButton" name="autoJava_pb">
            <property name="text">
             <string>&amp;Auto</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="javaPath_showPb">
            <property name="text">
             <string>Show...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="javaArg_label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>extra Java arguement:</string>
            </property>
            <property name="buddy">
             <cstring>javaArg_te</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1" rowspan="2">
           <widget class="QTextEdit" name="javaArg_te"/>
          </item>
          <item row="1" column="0">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>playerName_le</tabstop>
  <tabstop>autoName_cb</tabstop>
  <tabstop>email_le</tabstop>
  <tabstop>password_le</tabstop>
  <tabstop>showPassword_pb</tabstop>
  <tabstop>corePath_le</tabstop>
  <tabstop>corePath_showPb</tabstop>
  <tabstop>width_sb</tabstop>
  <tabstop>height_sb</tabstop>
  <tabstop>maxMem_sb</tabstop>
  <tabstop>minMem_sb</tabstop>
  <tabstop>javaPath_le</tabstop>
  <tabstop>autoJava_pb</tabstop>
  <tabstop>javaPath_showPb</tabstop>
  <tabstop>javaArg_te</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Preference</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Preference</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
